homepageapp/templates/registration/signin.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'homepageapp/base.html' %}
+
+
+{% block content %}
+
+<div class="container">
+    {% for message in messages %}
+        <div class="alert alert-{{ message.tags }} alert-dismissible fade show" role="alert">
+        <strong>Message: </strong> {{ message }}
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+        </button>
+        </div>
+    {% endfor %}
+
+    <h3>Log in</h3>
+
+    <form action="/signin/" method="post">
+    {% csrf_token %}
+
+
+        <label for="">Username</label>
```

Comment:
{{form.as_p}}

Reply:
fixed!

Reply:
hmm does not look like. 
If you use form then you don't need any other fields right?
The only thing you need is submit.

Reply:
I also don't see forms.py file, can you point me to it?